src/reducers/index.js

```diff
@@ -18,13 +19,14 @@ import users from './userReducer';
 
 const entities = combineReducers({
   catalogs,
+  clusterLabels,
   clusters,
   credentials,
   invitations,
+  nodePools,
   organizations,
   releases,
   users,
-  nodePools,
```

Comment:
Sorted a-z. Or this the order intentional?

Reply:
Let's hope not 😬